tests/rest/client/v2_alpha/test_auth.py

```diff
@@ -133,47 +133,6 @@ def test_fallback_captcha(self):
         # We're given a registered user.
         self.assertEqual(channel.json_body["user_id"], "@user:test")
 
-    def test_legacy_registration(self):
```

Comment:
This test becomes redundant with the test modified below since registration is not special anymore.